index.html

```diff
@@ -3,13 +3,20 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | Zaw (zkhing)</title>
     <link href="style.css" rel="stylesheet" />
     <link rel="icon" href="/favicon.ico" type="image/x-icon" />
   </head>
 
   <body>
```

Comment:
You should state somewhere - maybe in a header  here - that your data is from TVMaze and link back to their website. Otherwise you are not respecting their licence terms: https://www.tvmaze.com/api#licensing